addons/TBMod_rhs/config.cpp

```diff
@@ -10,7 +10,9 @@ class CfgPatches
 
         requiredAddons[] = {
             "TBMod_main",
-            "rhsusf_main"
+            "rhsusf_main",
+            "TBMod_skins",
+            "ace_advanced_ballistics"
```

Comment:
das kann wieder raus, wird jetzt ordentlich in TBmod_main required

Reply:
Hast du das bereits gemacht? Falls nicht, kannst du es direkt übernehmen :)

Reply:
wird später in #248 gemacht